app/controllers/articleController.js

```diff
@@ -116,6 +117,54 @@ export default class ArticleController {
     }
   }
 
+  /**
+   * Fuzzy searches for articles based on title
+   *
+   * @param {Request} req
+   * @param {Response} res
+   */
+  static async searchByTitle(req, res) {
+    try {
+      const search = req.query.search;
+
+      const fields = ["title"];
+
+      var results = await ArticlesAccessor.searchArticles(search, fields);
+
+      res.status(200).json(results);
```

Comment:
What do these results look like? Is it a list of articles, is it a mapping of scores to articles? Maybe add some outgoing validation here API model or JSON schema (I will merge JSON Schema in)